generators/server/templates/src/main/resources/logback-spring.xml.ejs

```diff
@@ -160,8 +163,7 @@
 <%_ } _%>
     <!-- jhipster-needle-logback-add-log - JHipster will add a new log with level -->
 
-    <!-- https://logback.qos.ch/manual/configuration.html#shutdownHook and https://jira.qos.ch/browse/LOGBACK-1090 -->
-    <shutdownHook class="ch.qos.logback.core.hook.DelayingShutdownHook"/>
+    <shutdownHook class="ch.qos.logback.core.hook.DefaultShutdownHook"/>
```

Comment:
Can be deleted since spring-boot 3 already define this by default

```suggestion
```

Reply:
@DanielFran I don't see it in https://github.com/spring-projects/spring-boot/blob/v3.0.0/spring-boot-project/spring-boot/src/main/resources/org/springframework/boot/logging/logback/defaults.xml.